### ISSUE TITLE
NVDA is announcing all the information present in the section before announcing the actual item where the focus is on.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#15342]
### Steps to reproduce:

1. Open Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.5.
2. Run NVDA
3. Tab to continue without code and press enter
4. press alt and use arrow keys to the tools option press enter
5. use arrow keys to the options, press enter.
6. tab to the search box and type "live"
7. use arrow keys and move the general under the "Live unit testing" press enter.
8. tab to the "restart test run---" checkbox in the main section and check the announcement of the screen reader.

### Actual behavior:
When pressing tab from the general section the main section, NVDA is announcing the parts of all the content present in the section as (property page  General,%,Default workspace root:,Memory Consumption,Cap memory for Live Unit Testing process at approximately,MB,Logging,Logging Level:) and at the last it is announcing the item where the focus is landed.
### Expected behavior:
When pressing tab from the general section the main section, NVDA should be announcing only the grouping information and the info about the item where the focus is landed as "General  grouping, Restart test run on document save  check box  checked.

### NVDA logs, crash dumps and other attachments:

https://github.com/nvaccess/nvda/assets/93735775/abdc0bc4-3dd8-45a2-9582-e16240928823

#### Does the issue still occur after restarting your computer?
Yes

**Comment by Adriani90** [comment]
This is because there is a property page added to the settings category which contains key words from the settings category. This is wrong design. I guess the implementation is something like this:
https://learn.microsoft.com/en-gb/previous-versions/visualstudio/visual-studio-2010/bb675676(v=vs.100)

So probably this property description was an attempt to describe the settings category resulting in over verbosity for the screen reader.

I saw a lot of dialogs in windows with such property descriptions and I am not sure what the purpose actually is.

**Comment by Adriani90** [comment]
@msftedad did you report this to your development team? If not, could you please do so? This issue needs an external fix by Microsoft.

**Comment by msftedad** [comment]
@Adriani90, We have verified this issue on our end and found that it is reproducible only with NVDA. The functionality is working as expected with other screen readers such as JAWS and Narrator.
